public_api/src/controllers/certos/public_graphql/request_graphql.rs

```diff
@@ -71,14 +74,27 @@ impl Showable<request::Request, RequestFilter> for Request {
       None
     };
     let mut entries = vec![];
-    for entry in db_entries {
+    for entry in db_entries.clone() {
       entries.push(vec![
         entry.attrs.id.to_string(),
         entry.attrs.state,
         entry.attrs.document_id.unwrap_or_else(|| "".to_string()),
         entry.attrs.email_callback_id.map(|i| i.to_string()).unwrap_or_else(|| "".to_string())
       ]);
     }
+
+    let mut admin_visited_count = 0;
+    let mut public_visit_count = 0;
+
+    for entry in db_entries {
```

Comment:
Map significa convertir un functor en otro. Hacer map de una colección a (), y que solo tiene por efecto mutar variables fuera del map, se escribe en un for. 
Si usaron map para reducir el nivel de nesting propongo cambiarlo así:
for entry in db_entries {
  let Some(document) = entry.document().await? else { continue; }
  let Some(l) = document.download_proof_link_scope().optional().await? else { continue; }
  if l.attrs.admin_visited { admin_visited_count += 1 };
  public_visit_count += l.attrs.public_visit_count;
}

Reply:
Modificado